scripts/generate-md-exports.mjs

```diff
@@ -58,7 +58,15 @@ async function uploadToCFR2(s3Client, relativePath, data) {
   return;
 }
 
-function taskFinishHandler({id, success, failedTasks}) {
+function taskFinishHandler(
+  {id, success, failedTasks, usedCacheFiles},
+  allUsedCacheFiles
```

Comment:
The way these parameters are used doesn't make much sense. We never seem to pass `usedCacheFiles` in the `data` object.

I think this logic needs some adjustments where `allUsedCacheFiles` is a global in the main/controller worker. It would also be probably more efficient to either use `union` on all these sets or just do the look up in order as looping over each file is not efficient.